staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2640,17 +2640,29 @@ func (i *IngressDescriber) describeIngressV1(ing *networkingv1.Ingress, events *
 		if def == nil {
 			// Ingresses that don't specify a default backend inherit the
 			// default backend in the kube-system namespace.
-			def = &networkingv1.IngressBackend{
+			defaultBackend := &networkingv1.IngressBackend{
 				Service: &networkingv1.IngressServiceBackend{
 					Name: "default-http-backend",
 					Port: networkingv1.ServiceBackendPort{
 						Number: 80,
 					},
 				},
 			}
-			ns = metav1.NamespaceSystem
+			defaultNamesapce := metav1.NamespaceSystem
```

Comment:
IMO, `defaultNamesapce` may confuse with `metav1.NamespaceDefault`.
We can just use `metav1.NamespaceSystem` directly or replace it with `systemNamespace`, WDYT?
```
_, err := i.client.CoreV1().Endpoints(metav1.NamespaceSystem).Get(context.TODO(), defaultBackend.Service.Name, metav1.GetOptions{})
```
```
systemNamesapce := metav1.NamespaceSystem
_, err := i.client.CoreV1().Endpoints(systemNamesapce).Get(context.TODO(), defaultBackend.Service.Name, metav1.GetOptions{})
```

Reply:
Agree and fixed. Thanks.